wpilibc/src/main/native/include/frc/simulation/SingleJointedArmSim.h

```diff
@@ -30,10 +30,13 @@ class SingleJointedArmSim : public LinearSystemSim<2, 1, 2> {
    * @param gearing            The gear ratio of the arm (numbers greater than 1
    *                           represent reductions).
    * @param armLength          The length of the arm.
-   * @param minAngle           The minimum angle that the arm is capable of.
-   * @param maxAngle           The maximum angle that the arm is capable of.
+   * @param minAngle           The minimum angle that the arm is capable of,
+   * with 0 being horizontal.
```

Comment:
For consistency with the rest of the comments, the overflowed text should be aligned (e.g. see the doc for the `gearing` parameter).